src/test/resources/self-hosted-dataset-ingest.json

```diff
@@ -1,2 +1,4 @@
-{"sample_name":"NA12878_exome","data_type":"exome","vcf_file_ref": "EXOME_VCF_FILE_ID","vcf_index_file_ref": ["EXOME_VCF_INDEX_FILE_ID"]}
-{"sample_name":"NA12878_wgs","data_type":"wgs","vcf_file_ref": {"description":"A downsampled wgs gVCF","mimeType":"text/plain","sourcePath":"gs://INGEST_BUCKET/selfHostedDatasetTest/vcfs/NA12878_PLUMBING_wgs.g.vcf.gz","targetPath":"/vcfs/downsampled/wgs/NA12878_PLUMBING.g.vcf.gz"},"vcf_index_file_ref": ["WGS_VCF_INDEX_FILE_ID"]}
\ No newline at end of file
+{"sample_name":"NA12878_exome1","data_type":"exome","vcf_file_ref": "${EXOME1_VCF_FILE_ID}","vcf_index_file_ref": ["${EXOME_VCF_INDEX_FILE_ID}"]}
```

Comment:
Note that this control file template is populated and uploaded to GS under a unique path within our test runs.  Modifying it here does not necessitate a reupload of any GS resources thought to be fixed.